### DISCUSSION TITLE
Keeping Espanso Updated

### DISCUSSION THREAD

**Discussion by fuzenco** [espanso/espanso#2148] [Q&A] [answered]
Does Espanso automatically update itself or do we have to download from Github when an update is released? I’m a designer, so not using a development process would be best for me.

**Comment by smeech** [comment]
Most people would update manually from the installation page at https://espanso.org/install/, I guess. It doesn't offer updates automatically.

**Reply by fuzenco** [reply]
Ok. Guess I’ll just keep an eye on GitHub releases notifications. Thanks.

**Comment by smeech** [comment, marked as answer]
If you want the newest code, you'll have to compile from the GitHub source. Otherwise, I expect we'll update the version number at the top of the docs when there's a new release - not that they're very frequent, although we should have one soon.
